feat: Kamil Ramocki task 1

kamilramocki opened the pull request:
I'm hopeful I've .gitignored everything I should've🙏🙏🙏

kamilramocki committed:
feat: add task 1 solution

czaja307 reviewed (changes requested):
pls fix CI errors (prettier, linter and test are currently failing)
if you have any difficulties during refactor, feel free to ask for help in the #techforum on discord server

kamilramocki committed:
fix: cl errors

kamilramocki committed:
fix: cl errors

kamilramocki committed:
fix: tests errors

kamilramocki committed:
fix: linter yapping??

kamilramocki committed:
fix: test lint yapping

kamilramocki committed:
fix: update ci.yml

it says that index.ts doesn't get along with prettier - but i don't even have index.ts

czaja307 reviewed (changes requested):
Pls do not disable the SolvroConfig - I think that the issue was caused because you set the base branch to main (which had some accidental changes then) instead of your `kamil-ramocki` branch. Try to push the revert to the prettier and we'll see then
